src/i-client-config.ts

```diff
@@ -134,19 +166,11 @@ export interface IClientConfig extends IBaseRequestConfig {
    */
   updateOnFetch?: ServingStoreUpdateStrategy;
 
-  /**
-   * A custom class to use for storing flag configurations.
-   * This is useful for cases where you want to use a different storage mechanism
-   * than the default storage provided by the SDK.
-   */
-  persistentStore?: IAsyncStore<Flag>;
+  // TODO: Add initial config (stringified IConfigurationWire) here.
```

Comment:
minor: it's better to add a proper `Configuration` class instead of passing untyped/unvalidated strings

Reply:
We have the client returning a stringified `IConfigurationWire` so devs don't need to worry about serializing the config string.

Reply:
In my ideal world, we have a `Configuration` class with `toString()` and `fromString()` methods. This way, devs still don't need to worry about serialization but they have a stronger hint at what they are expected to pass in

Reply:
There's a lot of refactoring to do around the configuration stores to properly plumb in Configuration and I can't wait.